src/web/components/ContributionSlot.tsx

```diff
@@ -0,0 +1,20 @@
+import { css } from '@emotion/react';
+
+export const ContributionSlot = () => {
+	return (
+		<div
+			css={css`
+				position: sticky;
+				top: 0;
+			`}
+		>
+			<a href="https://support.theguardian.com/uk/subscribe/digital?acquisitionData=%7B%22componentType%22%3A%22ACQUISITIONS_OTHER%22%2C%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22shady_pie_open_2019%22%2C%22componentId%22%3A%22shady_pie_open_2019%22%7D&INTCMP=shady_pie_open_2019">
```

Comment:
Any chance we can change the params here to reflect that this is a conversion from this, new, component?

Reply:
If you mean that maybe we should update the tracking params to distinguish this component from the one in frontend then I'm currently checking with the optimisation team whether they would like this or not. Will update here once I get an answer.

Reply:
I was suggesting changing shady_pie_open_2019 to shady_pie_open_2021 for example, so we can see in the recorded conversions how many people actually clicked though this newly implemented component.

Reply:
Ok, just got a reply back it would be better to leave it the same as it will then feed into all the existing reports and dashboards. From a business point of view the difference between the platforms is not really relevant.